key.ui/src/main/java/de/uka/ilkd/key/gui/MainWindow.java

```diff
@@ -85,10 +85,6 @@ public final class MainWindow extends JFrame {
      * SequentView for the current goal
      */
     public final CurrentGoalView currentGoalView;
-    /**
-     * The menu for the SMT solver options
-     */
-    public final JMenu smtOptions = new JMenu("SMT Solvers...");
```

Comment:
I guess this field was unused. And why do we have public fields in `MainWindow`?

Reply:
Yes, the field was not used at all. I have made the other public fields private.